runtime/src/proofs.rs

```diff
@@ -17,6 +18,10 @@ impl Proof {
             sorted_hashes,
         }
     }
+
+    pub fn get_leaf_hash(&self) -> H256 {
```

Comment:
can we make `leaf_hash` field in proofs public instead ?

Reply:
Yes, we can. I implemented it this way before. Then I found in anchor mod, a get_doc_root provided instead of setting data member public. I think it is better to following the team coding style.